dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -811,7 +811,7 @@ def _correct_bias_skewness(match_count: int, biased_skewness: float) -> float:
             )
             return np.nan
 
-        skewness = (
+        skewness: float = (
```

Comment:
as above

Reply:
Updated `_correct_bias_skewness` to return `Union[float, np.float64]`